### DISCUSSION TITLE
📷 Screenshots for ppl who wonder how it looks before download.

### DISCUSSION THREAD

**Discussion by ZeroCool22** [smy20011/dreambooth-gui#69] [General]
![Screenshot_4](https://user-images.githubusercontent.com/13344308/203261990-a1920337-1687-4503-84bd-ffcd599d3c42.jpg)
![Screenshot_3](https://user-images.githubusercontent.com/13344308/203261994-f541593a-65e1-4a4e-9acc-6a05d5ce310d.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/13344308/203261997-cc3c8456-a7fc-4162-b4fd-d953b4c5fbcd.jpg)

**Comment by smy20011** [comment]
Thank you!
